ICAFIX/scripts/functionhighpassandvariancenormalize.m

```diff
@@ -119,7 +121,9 @@ function functionhighpassandvariancenormalize(TR,hp,fmri,WBC,varargin)
         save_avw(reshape(confounds',size(confounds,2),1,1,size(confounds,1)),[fmri hpstring '.ica/mc/prefiltered_func_data_mcf_conf_hp'],'f',[1 1 1 TR]);
         
         cts=detrendpoly(cts',hp)';
-        save_avw(reshape(cts,ctsX,ctsY,ctsZ,ctsT),[fmri hpstring '.nii.gz'],'f',[1 1 1 TR]);
+        ctsfull=single(zeros(ctsX*ctsY*ctsZ,ctsT));
+	      ctsfull(ctsmask,:)=cts;
+        save_avw(reshape(ctsfull,ctsX,ctsY,ctsZ,ctsT),[fmri hpstring '.nii.gz'],'f',[1 1 1 TR]); clear ctsfull;
```

Comment:
Not sure if `clear ctsfull` is necessary or beneficial here (or in the later locations), since we need that same amount of memory available for `cstfull` later.